vmigration_helper/management/commands/migration_rollback.py

```diff
@@ -48,24 +48,23 @@ def add_arguments(self, parser):
         )
 
         parser.add_argument(
-            '--dry-run',
-            action='store_true',
-            help=f'Show the commands that would have run but do not run them'
+            "--migrate-cmd",
+            default=MIGRATE_COMMAND,
+            help=f'The migration command template (accepts {{app}} and {{name}}). Default is: "{MIGRATE_COMMAND}"',
```

Comment:
VS Code's settings formatted these to be more compatible with `black`. I kept the changes I added to be minimal but if it's a problem, I can revert all of these style changes.

Reply:
Never mind, I reverted those.